libs/util/shared/src/lib/models/search.model.ts

```diff
@@ -13,6 +13,12 @@ export interface StateConfigFilters {
   elastic?: any
 }
 
+export interface CatalogOrganisation {
```

Comment:
why not just `Organisation` just a question.

Reply:
My first thought was to easier differentiate it from `MetadataConctact.organisation` which is a `string` and I was hesitating putting the smart component into the `catalog` lib. Maybe `Organisation` makes more sense in the end.